### ISSUE TITLE
WebComponentList not initialized when annotation @FindAll is used

### ISSUE THREAD

**Issue by EbisuGit** [webdriverextensions/webdriverextensions#112]
I found out that if you have
```
@FindAll({ ... })
WebComponentList<YourComponent> components;
```
Components are not initialized. I looked up the code and it's probably this:
WebDriverExtensionFieldDecorator.java lines 150-153
```
if (field.getAnnotation(FindBy.class) == null
    && field.getAnnotation(FindBys.class) == null) {
  return false;
}
```

**Comment by andidev** [comment]
I think you may be right. The was based on seleniums [DefaultFieldDecorator](https://github.com/SeleniumHQ/selenium/blob/master/java/client/src/org/openqa/selenium/support/pagefactory/DefaultFieldDecorator.java#L90) a long time ago and most probably this annotation was added after this project was created.

Since this framework uses the seleniums DefaultFieldDecorator modifying the if case you are mentioning to also contain the FindAll will work.

**Comment by andidev** [comment]
Released in version 3.8.1
